packages/cdk-base-cf-acm-r53/lib/index.ts

```diff
@@ -87,14 +87,8 @@ export class CdkBaseCfAcmR53 extends cdk.Construct {
           httpStatus: 404,
           responseHttpStatus: 200,
           ttl: cdk.Duration.minutes(1),
-          responsePagePath: "/index.html"
-        },
-        {
-          httpStatus: 403,
-          responseHttpStatus: 200,
-          ttl: cdk.Duration.minutes(1),
-          responsePagePath: "/index.html"
-        },
+          responsePagePath: "/beta/index.html"
```

Comment:
This isn't the right place for this. The /beta path is specific to the one distribution that this was for.